ibis/backends/tests/test_numeric.py

```diff
@@ -101,7 +101,10 @@ def test_isnan_isinf(
         expected = backend.default_series_rename(expected)
         backend.assert_series_equal(result, expected)
     else:
-        assert result == expected
+        try:
```

Comment:
why this change? can you simply change the expected or make a dask specific test? catching an exception in a test is generally a no-no as it hides things

Reply:
This touches on a general issue I had with tests where many of them implicitly assume you're running pandas in the test (see anything with `pandas` in https://github.com/ibis-project/ibis/issues/2553#issuecomment-744035718). 

Here - I wanted to support equality for `dd.Series` objects without adding an explicit dependency on `dask` for running the general tests. 

I could change this to something like `if 'dd' in locals() and isinstance(expected, dd.Series):`

Reply:
However - it seems that `ibs/backends/tests` now assumes you have requirements for all envs installed. Is that intended?

Reply:
I think no, this is something that @datapythonista removed, are you rebased on master (which reverted the last change).

Reply:
Rebased now. So given that I don't necessarily want a dask dependency to run these tests, maybe these checks would be better than the exception?

```
 "dask" in str(type(df)
```
or

```
hasattr(df, "compute")
````